packages/contracts/src/utils.ts

```diff
@@ -154,6 +154,10 @@ export const add0x = (str: string): string => {
  * @param types The types of the variables in the `Result`. This can be retrieved from an
  * `ethers.Interface` object.
  * @param values The `Result` to convert.
+ *
+ * @returns The converted result. The returned object/array will not contain strings instead of
+ * BigInt values to ensure that `JSON.stringify` and `JSON.parse` can be called on it without
+ * causing an error.
```

Comment:
We now use this function (`recursivelyConvertResult`) to decode constructor and function arguments so that we can display them in the preview. This PR makes a slight revision to this function so that it never returns BigInts, which makes it safe to `JSON.stringify` the arguments in the preview